src/dst/CMakeLists.txt

```diff
@@ -62,6 +64,8 @@ target_link_libraries(dst
   PUBLIC
     utl
     OpenSTA
+    tcl
+    ${ZLIB_LIBRARIES}
```

Comment:
Is this going to be fixed in the next sta update?

Reply:
I think this is already fixed but let me verify that before I undo this change.